requirements.txt

```diff
@@ -1,3 +1,89 @@
-Flask
-gunicorn
-werkzeug
+attrs==15.2.0
+BeautifulSoup==3.2.1
+blinker==1.3
+certifi==2017.4.17
+chardet==3.0.4
+Cheetah==2.4.4
+click==6.7
+codecov==2.0.9
+configglue==1.1.2
+configobj==5.0.6
+coverage==4.4.1
+cryptography==1.2.3
+debtags==2.0
+defer==1.0.6
+dirspec==13.10
+distro-info===0.14build1
+dropbox==7.3.1
+duplicity==0.7.6
+ecdsa==0.13
+enum34==1.1.2
+feedparser==5.1.3
+first==2.0.1
+Flask==0.12.2
+gitdb2==2.0.2
+GitPython==2.1.3
+gunicorn==19.7.1
+gyp==0.1
+httplib2==0.9.1
+idna==2.5
+ipaddr==2.1.11
+ipaddress==1.0.16
+IPy==0.83
+itsdangerous==0.24
+Jinja2==2.9.6
+keyring==7.3
+launchpadlib==1.10.3
+lazr.restfulclient==0.13.4
+lazr.uri==1.0.3
+libvirt-python==1.3.1
+lockfile==0.12.2
+lxml==3.5.0
+Mako==1.0.3
+MarkupSafe==0.23
+nemo-emblems==3.2.3
+netifaces==0.10.4
+oauth==1.0.1
+oauthlib==1.0.3
+oneconf==0.3.9
+PAM==0.4.2
+paramiko==1.16.0
+pexpect==4.0.1
+Pillow==3.1.2
+pip-tools==1.9.0
+piston-mini-client==0.7.5
+protobuf==2.6.1
+ptyprocess==0.5
+pyasn1==0.1.9
+pyasn1-modules==0.0.7
+pycrypto==2.6.1
+pycups==1.9.73
+pycurl==7.43.0
+Pygments==2.1
+pygobject==3.20.0
+pyinotify==0.9.6
+PyJWT==1.3.0
+pyOpenSSL==0.15.1
+pyparted==3.10.7
+pyserial==3.0.1
```

Comment:
The list should look like this with the == signs but it must be smaller. I guess, you used pip freeze to geenrate the list.